aea/helpers/env_vars.py

```diff
@@ -68,6 +91,11 @@ def replace_with_env_var(
 
     if var_name in env_variables:
         var_value = env_variables[var_name]
+    elif type_str == "list":
```

Comment:
Does this mean it's going to "unfold" all env variables declared as list and use VAR_NAME_0 _1, etc... ?
What about if the user wants a list?

Reply:
It will take care of that, check the example script I have shared above